tests/helpers.js

```diff
@@ -450,17 +476,46 @@ function callWithTimeout(fn, timeout) {
 
 /**
  * Returns a Jest describe function that is skipped if the current platform is not the given one.
- * @param platform
- * @returns describe(.skip) function
  */
 function describeSkipOnPlatform(platform) {
 	return process.platform === platform ? global.describe.skip : global.describe;
 }
 
-module.exports.FuzzTestBuilder = FuzzTestBuilder;
-module.exports.FuzzingExitCode = FuzzingExitCode;
-module.exports.JestRegressionExitCode = JestRegressionExitCode;
-module.exports.WindowsExitCode = WindowsExitCode;
-module.exports.makeFnCalledOnce = makeFnCalledOnce;
-module.exports.callWithTimeout = callWithTimeout;
-module.exports.describeSkipOnPlatform = describeSkipOnPlatform;
+async function getFiles(dir) {
+	const result = [];
+	const files = await fs.promises.readdir(dir);
+	for (const file of files) {
+		const filepath = path.join(dir, file);
+		result.push(filepath);
+		if ((await fs.promises.stat(filepath)).isDirectory()) {
+			result.push(...(await getFiles(filepath)));
+		}
+	}
+	return result;
+}
+
+async function fileExists(path) {
+	return !!(await fs.promises.stat(path).catch((e) => false));
+}
+
+async function cleanCrashFilesIn(path) {
+	for (const file in await getFiles(path)) {
+		if (file.includes("crash-")) {
```

Comment:
Crash files in the test directory should not be a problem when fuzzing on CLI (other than being annoying). This is only relevant when running regression mode after the fuzzing mode has found a crash and has not deleted it.
What if my test is called `crash-`? It isn't now, but it might be in the future.
As a compromise, I suggest updating this with a pattern that looks for the SHA1 digest  in the filename:
```suggestion
		if (file.match(/crash-[0-9a-f]{40}/)) {
```